backend/apps/core/migrations/0001_initial.py

```
@@ -0,0 +1,24 @@
+# Generated by Django 5.0.14 on 2026-07-09 02:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)